libs/component-sample/src/lib/feature/side-menu-filters-sample/side-menu-filters-sample.component.html

```diff
@@ -1,7 +1,10 @@
 <sds-page>
     <sds-page-options>
         <sds-toolbar #toolbar="sdsToolbar" class="top-205">
-            <sds-filters [form]="form" [fields]="fields" [model]="model" (filterChange)="filterChange$.next($event)"></sds-filters>
+            <sds-accordion-item>
```

Comment:
when it's not adjacent to the sidenav should we omit the sds-accordion-item element?

Reply:
Yes, we can omit the sds-accordion-item element, this is a sample example to show that now accordion item is not part of sam-filters